frontend/components/site/SiteSettings/EnvironmentVariables.jsx

```diff
@@ -81,7 +82,7 @@ class EnvironmentVariables extends Component {
     return (
       <>
         <h3>Environment Variables</h3>
```

Comment:
This is off topic for this PR, but with the addition of the trash icon for deleting scan rules, should we update the environment variable table to use the trash icon button to delete an environment variable?